sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonPlugin.java

```diff
@@ -34,18 +36,53 @@
 public class PythonPlugin extends SonarPlugin {
 
   public static final String FILE_SUFFIXES_KEY = "sonar.python.file.suffixes";
+  private static final String CATEGORY = "Python";
 
   @Override
   public List getExtensions() {
+
     return ImmutableList.of(
+
       PropertyDefinition.builder(FILE_SUFFIXES_KEY)
         .name("File Suffixes")
         .description("Comma-separated list of suffixes of Python files to analyze.")
-        .category("Python")
+        .category(CATEGORY)
         .onQualifiers(Qualifiers.PROJECT)
         .defaultValue("py")
         .build(),
 
+      PropertyDefinition.builder(PythonCoverageSensor.REPORT_PATH_KEY)
+        .name("Path to coverage report(s)")
+        .description("Path to coverage reports. Ant patterns are accepted for relative path. The reports have to conform to the Cobertura XML format.")
+        .category(CATEGORY)
```

Comment:
@vilchik-elena It may be clearer to group some properties under a "Tests and Coverage" category, [the same way we do in JavaScript](https://github.com/SonarSource/sonar-javascript/blob/master/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/JavaScriptPlugin.java).